<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuBench</class>
 <widget class="QDialog" name="MenuBench">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>154</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_menu">
    <item>
     <widget class="QCommandLinkButton" name="btn_sys">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_graph">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_bench">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="btn_net">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_stress_cpu">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU Stress test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stress_ram">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>RAM Stress test</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>701</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_graph_cpu"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>701</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_graph_ram"/>
  </widget>
  <widget class="QSpinBox" name="spin_timeout">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>70</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>73</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Timeout (s)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>13</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CPU number</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_cpu_nb">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Block size (B)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_ram_block_size">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>7</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_ram_block_nb">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>37</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Blocks number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
